JSDemos/Demos/SelectBox/Overview/Angular/app/app.component.html

```diff
@@ -40,6 +46,7 @@
     <div class="dx-field-value">
       <dx-select-box
         [dataSource]="data"
+        [inputAttr]="{ 'aria-label': 'Product From DS' }"
```

Comment:
```suggestion
        [inputAttr]="{ 'aria-label': 'Product ID' }"
```
plz, update in all frameworks